src/store/modules/geometry/mutations.js

```diff
@@ -34,12 +34,12 @@ export function initGeometry(state, payload) {
 }
```

Comment:
Mostly fixed code styling in this file. Only new code is the method at the bottom, `updateClonedGeometry`